infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationFSM.java

```diff
@@ -349,7 +349,7 @@ void transition(LogReplicationState from, LogReplicationState to) {
         to.onEntry(from);
     }
 
-    public void setSiteConfigID(long siteConfigID) {
+    public void setTopologyConfigId(long siteConfigID) {
         this.siteConfigID = siteConfigID;
         snapshotReader.setSiteEpoch(siteConfigID);
```

Comment:
now that you are unifying the topologyConfigId, please change it also on. the snapshot and logEntry reader to setTopologyConfigId.

Reply:
Done.